Nodsoft.WowsReplaysUnpack.ExtendedData/Models/ReplayPlayer.cs

```diff
@@ -84,6 +84,13 @@ public record ReplayPlayer
 	public bool TtkStatus { get; set; }
 
 	public Lazy<ShipData?> ShipData { get; }
+	
+	
+	/*
+	 * FIXME: This is a hack to get replays to parse correctly post 0.11.11+.
+	 * No idea what it is for now, my assumption was on the new Key Target / Cookie feature.
+	 */
+	public object IsCookie { get; set; }
```

Comment:
Since this seems to trigger a nullability warning, it may be a good idea to actually address this even if it is just by assigning a default value with `IsCookie = default!`